samples/ftd_configuration/physical_interface_with_default_dhcp.yml

```diff
@@ -0,0 +1,52 @@
+- hosts: all
+  connection: httpapi
+  vars:
+    iface_name: "inside"
+    iface_hardware_name: "GigabitEthernet0/1"
+  tasks:
+    - name: Get List of DHCP servers
+      ftd_configuration:
+        operation: "getDHCPServerContainerList"
+        register_as: dhcp_containers_list
+
+    - name: Execute 'editDHCPServerContainer' operation
+      ftd_configuration:
+        operation: "editDHCPServerContainer"
+        data:
+            name: "{{ dhcp_containers_list.0.name }}"
+            version: "{{ dhcp_containers_list.0.version }}"
+            autoConfig: "{{ dhcp_containers_list.0.autoConfig }}"
+            primaryDNS: "{{ dhcp_containers_list.0.primaryDNS }}"
+            secondaryDNS: "{{ dhcp_containers_list.0.secondaryDNS }}"
+            primaryWINS: "{{ dhcp_containers_list.0.primaryWINS }}"
+            secondaryWINS: "{{ dhcp_containers_list.0.secondaryWINS }}"
+            servers: '{{ dhcp_containers_list.0.servers | selectattr("interface.name", "ne", iface_name) | list }}'
+            interface: "{{ dhcp_containers_list.0.interface }}"
+            type: "{{ dhcp_containers_list.0.type }}"
+        path_params:
+            objId: "{{ dhcp_containers_list.0.id }}"
+
+    - name: Setup Inside Interface with static IP
```

Comment:
Do we need to update an interface? I guess nothing changes in its configuration, right?

Reply:
make sens. removed this step